src/components/ec-phone-number-input/ec-phone-number-input.story.js

```diff
@@ -0,0 +1,61 @@
+import { action } from '@storybook/addon-actions';
+import EcPhoneNumberInput from './ec-phone-number-input.vue';
+
+export default {
+  title: 'Phone Number Input',
+  component: EcPhoneNumberInput,
+};
+
+export const basic = (args, { argTypes }) => ({
+  components: { EcPhoneNumberInput },
+  props: Object.keys(argTypes),
+  data() {
+    return {
+      model: {},
```

Comment:
Yes the way this work now is you initialiaze the v-model. 

In reality you will not need all the (get) model values to initialize it so the following 3. Area code, Icon Path and phone number.
```      
      phoneNumberInputModel: {
        country: {
          value: '+44',
          iconPath: '/static/img/country-flags/100/gb.png',
        },
        phoneNumber: '123456789',
      },
      
Value + Phonenumber is the values we want to initialize so we will know them. 
Icon path we will need to compute it. We need to know our repo (EBO) node modules setup to know where the img will get stored plus the iso code.

Reply:
Well, when a user has not set up the 2FA, we will need to by default select the country without having any more information (at that point we don't have the phoneNumber). Is it enough only haaving the country iso code 2? I think we will need to get the prefix number (+44) when requesting the area codes data to pass at least the country object (value and iconPath).

Reply:
@jmesao so you want to have the option to only render the flag?

Reply:
The thing is que we are going to have externally the ISO Code 2 of the country that needs to be selected by default initially. Can we do this with the current status of the component? How?

Reply:
Ok, after a conversation with Stavros I understand that it must be done on EBO side, and use the v-model structure to define the country selected by default

Reply:
@jafcalvente @jmesao Had another looks with @mcibique today and now you can pass it like that 
```
      phoneNumberInputModel: {
        country: { areaCode: '+44', text: 'United Kingdom', countryCode: 'GB' },
        phoneNumber: '123456789',
      },
 ```
 
 or to make it simpler assuming `countries` is the countries list you just pass the whole item from the list.
 ```
   country: countries[0],
 ```
 
 This will preselect it with the flag this time. Let me know if something else